Assets/Samples/GamepadMouseCursor/VirtualMouseInput.cs

```diff
@@ -2,6 +2,8 @@
 using UnityEngine.InputSystem.LowLevel;
 using UnityEngine.UI;
 
+////TODO: respect cursor lock mode
```

Comment:
Should we be capturing this stuff somewhere else? Maybe a favro board?

Reply:
Most of the bigger tickets are captured in our [JIRA backlog](https://unity3d.atlassian.net/secure/RapidBoard.jspa?rapidView=87&view=planning.nodetail). However, in general I find that keeping things *with* the source is indeed the most effective way to track stuff. These are the TODOs and REVIEWs that I find myself coming back to over and over whereas the JIRA stuff is out of view and slowly decaying most of the time. Like with docs, I find that the further you go from the source, the more useless and outdated the stuff becomes.